include/swift/SILOptimizer/Utils/Differentiation/Common.h

```diff
@@ -1,4 +1,4 @@
-//===--- Differentiation.h - SIL Automatic Differentiation ----*- C++ -*---===//
+//===--- Common.h - Automatic Differentiation Common Utils ----*- C++ -*---===//
```

Comment:
What makes derivative lookup not "common"?

Reply:
`Common.h` is just a temporary catch all file. I plan to add more files to this directory. Eventually, code in `Common.h` will get refactored out into other files and will get removed.